.github/workflows/maven.yml

```diff
@@ -422,12 +422,16 @@ jobs:
       - name: Run benchmark
         # Note: user/pass should match those used in sample configs.
         run: |
-          java -jar benchbase.jar -b ${{matrix.benchmark}} -c config/sqlserver/sample_${{matrix.benchmark}}_config.xml --create=true --load=true --execute=true --json-histograms results/histograms.json
+          java -jar benchbase.jar -b ${{matrix.benchmark}} -c config/sqlserver/sample_${{matrix.benchmark}}_config.xml -im 1000 -mt advanced --create=true --load=true --execute=true --json-histograms results/histograms.json
           # FIXME: Reduce the error rate so we don't need these overrides.
           if [ ${{matrix.benchmark}} == tatp ]; then
               ERRORS_THRESHOLD=0.05
           fi
           ./scripts/check_histogram_results.sh results/histograms.json $ERRORS_THRESHOLD
+          if ![ -d "./results/monitor" ]; then
```

Comment:
@bpkroth : Is this what you had in mind?

Reply:
Close.  That's just checking that the directory was created though, not that the contents of it were written somewhat sanely.

Also, I think we should enable this check for both Postgres and SqlServer if they're both supported.

Reply:
Replaced with a count for the number of files which should be at least 3 in the current setup (single + repeated query events, system events).

For Postgres to use advanced monitoring, you'd need to deploy an extension first. Is that possible in this pipeline?

Reply:
Also: Any idea why the docker test is failing? I didn't touch it and the error message is not helpful.

Reply:
> For Postgres to use advanced monitoring, you'd need to deploy an extension first. Is that possible in this pipeline?

Probably - can you point me at the extension?  I can probably help with that.